src/options.tsx

```diff
@@ -177,12 +177,6 @@ class SvgMappings extends React.PureComponent<PanelOptionsEditorProps<SVGIDMappi
 
 export const optionsBuilder = (builder: PanelOptionsEditorBuilder<ACESVGOptions>) => {
   return builder
-    .addBooleanSwitch({
```

Comment:
these do not appear to do anything... can we remove them?

Reply:
Yes, monaco seems to have much better performance than the other editor. And the default implementation doesn't really give you a clean option to turn off autocomplete.